YoCode/Program.cs

```diff
@@ -81,10 +81,7 @@ private static void Main(string[] args)
 
             var results = new Results(evidenceList, appSettingsBuilder.GetWeightingsPath());
 
-            //var fd = new FileDifference(dir.ModifiedTestDirPath);
-
-
-            compositeOutput.PrintFinalResults(evidenceList.OrderBy(a => a.FeatureTitle),
+            compositeOutput.PrintFinalResults(evidenceList.OrderBy(a => FeatureTitleStorage.GetFeatureTitle(a.Feature)),
```

Comment:
I feel like we should define an order for our checks rather than relying on alphabetical sorting, but I don't want to do anything about that now, it's not worth it. Just seeing `GetFeatureTitle` used here made me realise it's a bit odd.

Reply:
I agree